### PR TITLE
params and policies configuration through profiles defined on startup

### PR DESCRIPTION
**Pull request by Alexey-N-Chernyshov**
Signed-off-by: Alexey-N-Chernyshov <alexey.n.chernyshov@gmail.com>

### Description of the Change
Add profiles for policies and parameters configuration.

### Possible Drawbacks 
- breaks naming conventions kConstants are not actually constants.
- introduces grobal variables 

### Alternate Designs *[optional]*
- create structs for policies and parameters and pass them instead of global vars.

### CODE REVIEW

File: core/const.cpp [hunk 1]
```diff
@@ -4,9 +4,97 @@
...
+#include "vm/actor/builtin/types/verified_registry/policy.hpp"
 
 namespace fc {
-  DEFINE(kConsensusMinerMinPower){BigInt{1} << 40};
+  using primitives::sector::RegisteredSealProof;
+
+  /**
+   * Class for profile name validation with boost::program_options
+   */
+  class Profile : public std::string {};
```
Comment (wer1st) [inline core/const.cpp:21]: Why not `using Profile = std::string`?

File: core/vm/actor/builtin/types/market/policy.hpp [hunk 2]
```diff
@@ -25,7 +25,7 @@ namespace fc::vm::actor::builtin::types::market {
                                           * TokenAmount{"1000000000000000000"}};
 
   /// The number of blocks between payouts for deals
-  constexpr EpochDuration kDealUpdatesInterval{fc::kEpochsInDay};
+  static EpochDuration kDealUpdatesInterval = fc::kEpochsInDay;
```
Comment (wer1st) [inline core/vm/actor/builtin/types/market/policy.hpp:28]: Why static? constexpr looks better

**Review by wer1st — APPROVED** [review]

File: core/const.cpp (hunk 1 above)
Comment (Alexey-N-Chernyshov) [reply core/const.cpp:21]: fixed

File: core/vm/actor/builtin/types/market/policy.hpp (hunk 2 above)
Comment (Alexey-N-Chernyshov) [reply core/vm/actor/builtin/types/market/policy.hpp:28]: It is not `const` anymore, changed to extern.

File: core/const.cpp (hunk 1 above)
Comment (Alexey-N-Chernyshov) [reply core/const.cpp]: Actually `using Profile = std::string` creates type alias, but not the new type required for validation.

**Review by turuslan — APPROVED** [review]

**Review by wer1st — APPROVED** [review]

**Pull request merged**